traceevents/src/main/java/com/tomtom/kotlin/traceevents/TraceLog.kt

```diff
@@ -54,11 +55,17 @@ interface TraceLog {
          */
         internal object DefaultLoggerToStdout : Logger {
             override fun log(logLevel: LogLevel, tag: String, message: String, e: Throwable?) {
-                val exceptionMsg = e?.message?.let { ", $it" } ?: ""
-                println("${LocalDateTime.now()}: [$logLevel] $tag: $message$exceptionMsg")
+                println("${createStdoutLogLine(logLevel, tag, message, e)}")
             }
         }
 
+        internal fun createStdoutLogLine(
+            logLevel: LogLevel, tag: String, message: String, e: Throwable?
+        ) =
+            "[${LocalDateTime.now()
+                .format(DateTimeFormatter.ISO_DATE_TIME)}]: [$logLevel] $tag: $message" +
+                if (e == null) "" else (", " + Tracer.formatThrowable(e, true))
```

Comment:
nit: I'd suggest the following formatting for consistency
```suggestion
            "[${LocalDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)}]: [$logLevel] " +
                "$tag: $message${e?.let { ", ${Tracer.formatThrowable(it, true)}" } ?: ""}"
```